src/test/scala/zio/codec/examples/CodecJsonTest.scala

```diff
@@ -37,12 +34,12 @@ object CodecJsonTest {
   val brace2: Codec[Char]   = char('}')
 
   val spacing: Codec[Unit] =
-    consume.filter(Set(' ', '\n', '\r')).rep.ignore(Chunk.empty)
+    consume.oneOf(' ', '\n', '\r').rep.ignore(Chunk.empty)
 
-  val digit: Codec[Char]             = consume.filter(Set('0', '1', '2', '3', '4', '5', '6', '7', '8', '9'))
-  val sign: Codec[Option[Char]]      = consume.filter(Set('-', '+')).option
+  val digit: Codec[Char]             = between('0', '9')
+  val sign: Codec[Option[Char]]      = consume.oneOf('-', '+').option
```

Comment:
What do you think about naming `between` as `charBetween` to be consistent with `char` ?